{"name": "test_loging[www-www-Invalid login, please try again]", "status": "passed", "parameters": [{"name": "username", "value": "'www'"}, {"name": "password", "value": "'www'"}, {"name": "exp", "value": "'Invalid login, please try again'"}], "start": 1737366447366, "stop": 1737366465687, "uuid": "da332c3d-a237-4bc8-b826-c6d71557f8a7", "historyId": "aab5a279da802be6f1beac27b50c7d44", "testCaseId": "afa662b2472d3f5290ca51cc6cca78cc", "fullName": "allureReport.TestCase#test_loging", "labels": [{"name": "severity", "value": "minor"}, {"name": "feature", "value": "login"}, {"name": "suite", "value": "allureReport"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "Satwinder"}, {"name": "thread", "value": "21612-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "allureReport"}]}